great_expectations/datasource/data_connector/runtime_data_connector.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         self,
         name: str,
         datasource_name: str,
-        execution_engine: Optional[ExecutionEngine] = None,
```

Comment:
Another runtime change.
`ExecutionEngine` doesn't appear to be optional at all. The supertype has it as a positional argument and raises an error if it's not provided.